subprojects/docs/src/docs/userguide/dependencyManagementTroubleshootingDependencyResolution.adoc

```diff
@@ -0,0 +1,97 @@
+// Copyright 2018 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[troubleshooting_dependency_resolution]]
+== Troubleshooting dependency resolution
+
+Managing large dependency graphs can be challenging. This section describes techniques for troubleshooting issues you might encounter in your project.
```

Comment:
Is it in the short term plan to update this content?

Reply:
Depends on how much time we'll want to spend on the overall dependency management chapter. I think rewriting the introduction is much more important. We have an issue for this portion on the board. From my perspective we need to touch all chapters really before calling it done.